addons/mail/static/src/models/res_users_settings.js

```diff
@@ -0,0 +1,33 @@
+/** @odoo-module **/
+
+import { registerModel } from '@mail/model/model_core';
+import { attr, many, one } from '@mail/model/model_field';
+
+/**
+ * Mirrors the fields of the python model res.users.settings.
+ */
+registerModel({
+    name: 'res.users.settings',
+    identifyingFields: ['id'],
+    fields: {
+        id: attr({
+            readonly: true,
+            required: true,
+        }),
+        is_discuss_sidebar_category_channel_open: attr(),
+        is_discuss_sidebar_category_chat_open: attr(),
+        push_to_talk_key: attr(),
+        use_push_to_talk: attr(),
+        user_id: one('User', {
+            inverse: 'resUsersSettings',
+        }),
+        voice_active_duration: attr(),
+        /**
+         * States the volume chosen by the current user for each other user.
+         */
+        volume_settings_ids: many('VolumeSetting', {
```

Comment:
I would expect this to be `res.users.settings.volumes`, a python model.

Reply:
For volume settings specifically though we should introduce it now, as it is directly in the scope of this PR.